proxy/src/console/provider/neon.rs

```diff
@@ -328,19 +330,24 @@ async fn parse_body<T: for<'a> serde::Deserialize<'a>>(
         info!("request succeeded, processing the body");
         return Ok(response.json().await?);
     }
+    let s = response.text().await?;
```

Comment:
`text()` does some extra stuff that `json()` doesn't do. Using `bytes()` should make it more equivalent. The `Bytes` type implements a nice utf8 aware debug impl, so it should still be readable.

```suggestion
    let s = response.bytes().await?;
```